spec/controllers/users_controller_spec.rb

```diff
@@ -165,20 +165,15 @@
 
   describe "DELETE destroy" do
     let(:user) { create(:user) }
-    before :each do
-      sign_in user
-    end
+    before { sign_in user }
 
     context "its own profile" do
-      it "destroys the requested user" do
-        expect {
-          delete :destroy, params: { id: user.to_param }
-        }.to change(User, :count).by(-1)
-      end
+      it "destroys the requested user and redirects to /community" do
```

Comment:
I know this violates the "one assertion per example" rule. But that's how I roll in controller/request/feature/functional tests because the setup of DB objects is too expensive to not reuse the existing setup.

Reply:
I am totally convinced ;-) 
In our first Dev Sync we talked about moving from controller tests to request tests. Would that help in this case?

Reply:
Not as long the "one assertion per example" mantra is still being held up; the test would look almost the same, only the test macros change a little. I did not rewrite this as a request spec (even though we decided to gradually move request specs in the first DevSync as you mentioned) since we already had an exisiting test and I think the move to request specs should be one big PR on its own.